runtime/BUILD.gn

```diff
@@ -168,3 +173,23 @@ executable("runtime_unittests") {
     ":runtime_unittests_common",
   ]
 }
+
+if (is_fuchsia) {
+  fuchsia_test_archive("runtime_tests") {
+    deps = [
+      ":runtime_unittests",
+      ":runtime_fixtures",
+    ]
+
+    binary = "runtime_unittests"
+
+    # Right now we need to manually specify all the fixtures that are
```

Comment:
Can you add a TODO/Issue here to make the fixtures mechanism do this for you automatically in the future?

Reply:
Filed https://github.com/flutter/flutter/issues/50294